.github/workflows/create-builds.yml

```diff
@@ -98,21 +100,23 @@ jobs:
       - name: Upload artifacts
         uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
         with:
-          name: ${{ matrix.os.name }}-${{ env.BUILD_SHA }}
+          name: builds-${{ matrix.os.name }}-${{ env.BUILD_SHA }}
           compression-level: 0
           overwrite: true
-          path: |
-            out/make/**/RELEASES
-            out/make/**/*.AppImage
-            out/make/**/*.deb
-            out/make/**/*.dmg
-            out/make/**/*.exe
-            out/make/**/*.nupkg
-            out/make/**/*.rpm
-            out/make/**/*.zip
+          if-no-files-found: error
+          path: out/distributables
 
-      - name: Publish application
-        if: ${{ inputs.APP_TYPE == 'production' }}
-        env:
-          GITHUB_TOKEN: ${{ steps.app-token.outputs.token }}
-        run: node --run forge:publish
+  upload_merged:
+    name: Upload merged artifacts
+    needs: builds
+    runs-on: ubuntu-latest
+    outputs:
+      artifact_id: ${{ steps.merge_and_upload.outputs.artifact-id }}
+    steps:
+      - name: Merge and upload artifacts
+        id: merge_and_upload
+        uses: actions/upload-artifact/merge@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: builds-${{ env.BUILD_SHA }}
+          delete-merged: true
+          compression-level: 0
```

Comment:
wondering if it would be preferable to do this in the release workflow instead of here. Being able to have matrix-separated artifacts for the create-builds workflow is kind of helpful for debugging and reusability.

Reason I didn't do it is is that it's harder to test that this generally works, as it would require me to run a "test" release workflow.